myia/utils/variables.py

```diff
@@ -50,3 +50,9 @@ def _is_c(n):
 Xs = SVar(Var())
 Ys = SVar(Var())
 Cs = SVar(constvar())
+
+
+__consolidate__ = True
+__all__ = [
+    'constvar',
```

Comment:
We might not want this, but all the other ones instead.

Reply:
opt/lib does use constvar.